openstack/compute/v2/extensions/simpletenantusage/requests.go

```diff
@@ -0,0 +1,72 @@
+package simpletenantusage
+
+import (
+	"time"
+
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+// Get returns simple tenant usage data about all tenants
+func Get(client *gophercloud.ServiceClient, opts GetOptsBuilder) pagination.Pager {
+	url := getURL(client)
+	if opts != nil {
+		query, err := opts.ToSimpleTenantUsageGetMap()
+		if err != nil {
+			return pagination.Pager{Err: err}
+		}
+		url += query
+	}
+	return pagination.NewPager(client, url, func(r pagination.PageResult) pagination.Page {
+		return SimpleTenantUsagePage{pagination.LinkedPageBase{PageResult: r}}
+	})
+}
+
+// GetTenant returns simple tenant usage data about a specific tenant
+func GetTenant(client *gophercloud.ServiceClient, tenantID string, opts GetOptsBuilder) pagination.Pager {
+	url := getTenantURL(client, tenantID)
+	if opts != nil {
+		query, err := opts.ToSimpleTenantUsageGetMap()
+		if err != nil {
+			return pagination.Pager{Err: err}
+		}
+		url += query
+	}
+	return pagination.NewPager(client, url, func(r pagination.PageResult) pagination.Page {
+		return SimpleSingleTenantUsagePage{pagination.SinglePageBase(r)}
+	})
+}
+
+// Options for fetching the tenant usage for a Tenant.
+// All int-values are pointers so they can be nil if they are not needed
+// you can use gopercloud.IntToPointer() for convenience
+type GetOpts struct {
+	// Set to 1 to get detailed (i.e. "server usages") tenant usage; 0 otherwise.
+	Detailed *bool `json:"detailed,omitempty"`
+
+	// The ending time to calculate usage statistics on compute and storage resources.
+	End time.Time `json:"end,omitempty"`
+
+	// The beginning time to calculate usage statistics on compute and storage resources.
+	Start time.Time `json:"start,omitempty"`
```

Comment:
There's a big issue with the `Start` and `End` fields: they never end up in the URL query. You can see this by inspecting the URL being build for the request or by looking at the Nova API service's log files -- start and end are never part of the URL. The reason is because the [`BuildQueryString`](https://github.com/gophercloud/gophercloud/blob/master/params.go#L303) function does not support time fields. 

There are two things we can do here:

1. Use `string`, though I don't think that's the best long-term solution
2. Change these to `*time.Time` and append `q` in `ToSimpleTenantUsageGetMap` with values if set:

```go
const TimeParseFormat = "2006-01-02T15:04:05.000000"
func (opts GetOpts) ToSimpleTenantUsageGetMap() (string, error) {
  q, err := gophercloud.BuildQueryString(opts)
  params, err := url.ParseQuery(q.RawQuery)
  if err != nil {
    return "", err
  }

  if opts.Start != nil {
    params.Add("start", opts.Start.Format(TimeParseFormat))
  }

  q = &url.URL{RawQuery: params.Encode()}
  return q.String(), err
}
``` 

@jrperritt thoughts on this one?